pom.xml

```diff
@@ -50,6 +50,7 @@
     <version.io.opentracing.contrib.tracerresolver>0.1.5</version.io.opentracing.contrib.tracerresolver>
     <version.io.opentracing.contrib.web-servlet-filter>0.1.1</version.io.opentracing.contrib.web-servlet-filter>
     <version.io.opentracing.contrib.okhttp3>0.1.0</version.io.opentracing.contrib.okhttp3>
+    <version.io.opentracing.contrib.jdbc>0.0.9-SNAPSHOT</version.io.opentracing.contrib.jdbc>
```

Comment:
Should not be dependent upon snapshot version - is there a dependency on a pending PR?

Reply:
No pending PR, but that's the version on the master branch for that contrib project https://github.com/opentracing-contrib/java-jdbc/blob/master/pom.xml#L21

Advise if I should just use 0.0.8 which has been published and should be the same.